src/index.ts

```diff
@@ -1 +1,4 @@
-export * from './Plot';
+import Plot from './Plot';
+import LineSeries from './components/LineSeries';
+
+export { Plot as default, LineSeries };
```

Comment:
Can we export everything with a name? It's not useful anyway to only import the default, so `import { Plot, LineSeries } from 'react-plot'` is simpler than `import Plot, { LineSeries } from 'react-plot'`